dev/2017/spring/breaking-down-debt/breaking-down-debt.js

```diff
@@ -0,0 +1,468 @@
+/* jshint ignore:start */
+/*!
+ * chartjs-plugin-deferred
```

Comment:
This is the deferred plugin for chartjs (it defers the animation until you scroll down) and not the chartjs lib.